test_scripts/Defects/4_5/1885_SDL_does_not_cut_off_unknown_RPC_and_or_param_from_PTU.lua

```diff
@@ -0,0 +1,158 @@
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups =
+    { "Base-4", "NewTestCaseGroup1" }
+end
+
+--[[ @ptuUpdateFuncParams: update table with unknown parameters for PTU
+--! @parameters:
+--! tbl - table for update
+--! @return: none
+--]]
+local function ptuUpdateFuncParams(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+        parameters = { "gps", unknownParameter }
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup2"] = VDgroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups =
+    { "Base-4", "NewTestCaseGroup1", "NewTestCaseGroup2" }
+end
+
+--[[ @SuccessfulProcessingRPC: Successful processing API
+--! @parameters:
+--! RPC - RPC name
+--! params - RPC params for mobile request
+--! interface - interface of RPC on HMI
+--! self - test object
+--! @return: none
+--]]
+local function SuccessfulProcessingRPC(RPC, params, interface)
+  local cid = common.getMobileSession():SendRPC(RPC, params)
+  common.getHMIConnection():ExpectRequest(interface .. "." .. RPC, params)
+  -- :Do(function(_,data)
+  --     self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  --   end)
+  :Do(function(_, data)
+    if RPC == "GetVehicleData" then
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {gps = gpsResponse})
+    elseif
+      RPC == "SubscribeVehicleData" then
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+      {gps = { dataType = "VEHICLEDATA_GPS", resultCode = "SUCCESS" }})
+    else
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end
+  end)
+  common.getMobileSession():ExpectResponse(cid,{ success = true, resultCode = "SUCCESS" })
+end
+
+--[[ @DisallowedRPC: Unsuccessful processing of API with Disallowed status
+--! @parameters:
+--! RPC - RPC name
+--! params - RPC params for mobile request
+--! interface - interface of RPC on HMI
+--! self - test object
+--! @return: none
+--]]
+local function DisallowedRPC(RPC, params, interface)
+  local cid = common.getMobileSession():SendRPC(RPC, params)
+  common.getHMIConnection():ExpectRequest(interface .. "." .. RPC)
+  :Times(0)
+  common.getMobileSession():ExpectResponse(cid, { success = false, resultCode = "DISALLOWED" })
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App1", common.registerApp)
+runner.Step("Activate App1", common.activateApp)
+
+runner.Title("Test")
+runner.Step("PTU update with unknown API", common.policyTableUpdate, { ptuUpdateFuncRPC })
+runner.Step("Check applying of PT by processing SendLocation", SuccessfulProcessingRPC,
```

Comment:
Why `SendLocation` RPC should be processed in case this RPC is not present in permissions update?

Reply:
@GetmanetsIrina, To verify that SDL cut off unknown RPC from PTU and that SendLocation RPC can be processed after PTU if it is present in permissions update